test/DependencyInjection.Tests/AspnetDiContainer/DependencyInjectionGrainTestsUsingASPNETDIContainer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.Hosting;
+using Orleans.TestingHost;
+using TestExtensions;
+using UnitTests.Grains;
+
+namespace DependencyInjection.Tests.AspnetDIContainer
+{
+    [TestCategory("DI"), TestCategory("BVT")]
+    public class DependencyInjectionGrainTestsUsingASPNETDIContainer : DependencyInjectionGrainTestRunner
+    {
+        public class Fixture : BaseTestClusterFixture
+        {
+            protected override void ConfigureTestCluster(TestClusterBuilder builder)
+            {
+                builder.Options.InitialSilosCount = 1;
+                builder.AddSiloBuilderConfigurator<TestSiloBuilderConfigurator>();
+                //Orleans would use ASP.NET DI container solution by default, so no need to configure ServiceProviderFactory here
+            }
+
+
+        }
+
+        public DependencyInjectionGrainTestsUsingASPNETDIContainer()
+            : base(new Fixture())
```

Comment:
This will create a new cluster per test. Please follow the standard pattern to use class fixtures